frontend/src/components/Selector.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+export class Selector extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div>
+        <h2>{this.props.name}</h2>
+        <div>
+          {Object.keys(this.props.documents).map(docClass => {
+            let buttonType
+            if (this.props.name === 'Selected') {
+              buttonType = 'X'
+            } else {
+              buttonType = '+'
+            }
+            if (this.props.documents[docClass] === this.props.name) {
+              return (
+                <div>
+                  {docClass} i{' '}
```

Comment:
I would mark the i as a placeholder with a comment so that we know to fix this later on when going through the code

Reply:
Changes Made.